cookbook/ui_customization/add_a_tab_to_product_edit_form.rst

```diff
@@ -0,0 +1,212 @@
+How to add a tab or a panel to the product edit form
+====================================================
+
+.. note::
+    #Add a note to the "How to work with the front in the PIM"
+
+The most common UI customization on the Akeneo PIM is to add a tab to the product edit form. With the new product edit form introduced in 1.4 (more details here#TODO add link) we splitted tabs in two concepts:
+
+ - Tabs hold all edit features on the entity (attributes edit, classification, associations)
+ - Panels hold all meta information about the entity (history, completeness, comments)
+
+Add a tab to the product edit form
+----------------------------------
+
+Let's say that we would like to add a custom tab to our product edit form in order to manage packages of the product.
+
+First, we need to create a Form extension in our bundle:
+
+.. code-block:: javascript
+    :linenos:
+
+    'use strict';
+    /*
+     * /src/Acme/Bundle/EnrichBundle/Resources/public/js/product/form/packages.js
+     */
+    define(['pim/form'],
+        function (BaseForm) {
+            return BaseForm.extend({
+                configure: function () {
+                    this.trigger('tab:register', {
+                        code: this.code,
+                        isVisible: this.isVisible.bind(this),
+                        label: 'Packages tab'
+                    });
+
+                    return BaseForm.prototype.configure.apply(this, arguments);
+                },
+                render: function () {
+                    this.$el.html('Hello world');
+
+                    return this;
+                },
+                isVisible: function () {
+                    return true;
+                }
+            });
+        }
+    );
+
+For now this is a pretty dumb extension, but this is a good start !
+
+Let's register this file in the `requirejs` configuration
+
+.. code-block:: yaml
+    :linenos:
+
+    # /src/Acme/Bundle/EnrichBundle/Resources/config/requirejs.yml
+
+    require:
+        paths:
+            pim/product-edit-form/packages: pimacme/js/product/form/packages
+
+Now that our file is registred into `requirejs` configuration, we can add this extension to the product edit form:
+
+.. code-block:: yaml
+    :linenos:
+
+    # /src/Acme/Bundle/EnrichBundle/Resources/config/form_extensions/product_edit.yml
+
+    extensions:
+        pim-product-edit-form-packages:                        # The form extension code (can be whatever you want)
+            module: pim/product-edit-form/packages             # The requirejs module we just created
+            parent: pim-product-edit-form-form-tabs            # The parent extension in the form where we want to be registred
+            targetZone: container
+            aclResourceId: pim_enrich_product_categories_view  # If you want to display this extension only under certain rights
```

Comment:
weird (a bit frenchy imho).
Could be nice to have a link (if the cookbook exists) to add rights in the PIM :)